lib-persistence-manager/go.mod

```diff
@@ -4,5 +4,6 @@ go 1.13
 
 require (
 	github.com/ODIM-Project/ODIM/lib-utilities v0.0.0-20200727091052-cb7db65624ce
+	github.com/go-redis/redis/v8 v8.0.0-beta.10
```

Comment:
Only the stable version of the packages should be used, shouldn't use beta and alpha version.

Reply:
Fixed it with latest release.